### DISCUSSION TITLE
Change to Firefox sidebar requires new translations

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/mdn-community#83] [Localization] [answered]
Hey everyone!

We just landed a pull request that introduced changes to the [Firefox sidebar macro](https://github.com/mdn/yari/blob/main/kumascript/macros/FirefoxSidebar.ejs). As a result of the changes, we need some of the new strings translated. The languages that require translations are:

- `fr` @yari-content-fr
- `bn-BD`
- `es` @yari-content-es
- `id`
- `it`
- `pl`
- `pt-BR` @yari-content-pt-br
- `ru` @yari-content-ru
- `zh-CN` @yari-content-zh
- `zh-TW`

You can [see the pull request here](https://github.com/mdn/yari/pull/5672) for reference.

When you have a moment, please have a look and open up a pull request with the new translations. You can ping me "@schalkneethling" for review. Thank you so much for all your amazing and hard work.

**Comment by SphinxKnight** [comment]
@mdn/yari-content-es @mdn/yari-content-fr @mdn/yari-content-pt-br @mdn/yari-content-ru @mdn/yari-content-zh

**Reply by schalkneethling** [reply]
Thank you @SphinxKnight! I tagged them wrong in the opening discussion 😄

**Reply by levensta** [reply]
@SphinxKnight it redirects to 404 🥲

**Reply by SphinxKnight** [reply]
@levensta I'm not sure what you're talking about here?

**Reply by levensta** [reply]
@SphinxKnight the links you attached redirect to 404 page. Maybe it should be and I don't have access. I would like to know who is a member of **@mdn/yari-content-ru** so that I know who to contact to check the pull requests related to translation

**Reply by cw118** [reply]
Hi @levensta, I don't think the mdn teams are visible to the public which explains the 404, but you can see review team members in [peers guidelines](https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#review-teams). I also see that the Firefox sidebar has already been translated to Russian (cf. mdn/yari#5699). I hope this helps :)

**Reply by levensta** [reply]
@cw118 oh, thank you! idk why I didn't to attention [this](https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#review-teams) 🙈

**Comment by ghost** [comment]
Hi @schalkneethling, can anyone do that? Or just for maintainers?

**Reply by schalkneethling** [reply]
Hey @victor0x16, this is open to anyone comfortable with translating to the languages listed. Thank you for your interest.

**Reply by ghost** [reply]
Thanks.

**Comment by myshov** [comment]
Hi @schalkneethling, Russian translation is done https://github.com/mdn/yari/pull/5699

**Reply by schalkneethling** [reply]
Thanks so much!

**Comment by Graywolf9** [comment]
Thank you so much for your guidance @schalkneethling 

There is the `es` PR: https://github.com/mdn/yari/pull/5911

**Reply by schalkneethling** [reply]
On my list. Thanks so much, @Graywolf9

**Comment by hochan222** [comment]
There is the `ko` PR:  https://github.com/mdn/yari/pull/7971/files

Have a nice day :)

**Comment by schalkneethling** [comment, marked as answer]
It looks like this one has been taken care of. Thank you, everyone! If I am mistaken, please feel free to reopen.
